Panic listing s3 buckets

One of my common prefixes appears to be named `/`. After slight modification, I get
```
panic: plugin.NewEntry: received name / containing a /

goroutine 743 [running]:
github.com/puppetlabs/wash/plugin.NewEntry(0x27aa356, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/michaelsmith/puppetlabs/wash/plugin/entryBase.go:50 +0x174
github.com/puppetlabs/wash/plugin/aws.newS3ObjectPrefix(0xc0072e0bc0, 0x11, 0x3b4712f, 0x1, 0xc0005b4798, 0x4)
	/Users/michaelsmith/puppetlabs/wash/plugin/aws/s3ObjectPrefix.go:25 +0x65
github.com/puppetlabs/wash/plugin/aws.listObjects(0x2aeaae0, 0xc0072e7cb0, 0xc0005b4798, 0xc0072e0bc0, 0x11, 0x0, 0x0, 0x0, 0xc0075a5cd8, 0x100d909, ...)
	/Users/michaelsmith/puppetlabs/wash/plugin/aws/s3Bucket.go:50 +0x3e5
github.com/puppetlabs/wash/plugin/aws.(*s3Bucket).List(0xc0074884d0, 0x2aeaae0, 0xc0072e7cb0, 0x10, 0xc003dd8d00, 0xc0005c7200, 0x8072, 0xc0004bc000)
	/Users/michaelsmith/puppetlabs/wash/plugin/aws/s3Bucket.go:105 +0x67
github.com/puppetlabs/wash/plugin.CachedList.func1(0xc0000da120, 0x5, 0x27b601e, 0x10)
	/Users/michaelsmith/puppetlabs/wash/plugin/cache.go:107 +0x5d
github.com/puppetlabs/wash/datastore.(*MemCache).GetOrUpdate(0xc00029f800, 0xc0073bc580, 0x31, 0x37e11d600, 0x0, 0xc0072e7ce0, 0x0, 0x0, 0x0, 0x0)
	/Users/michaelsmith/puppetlabs/wash/datastore/cache.go:80 +0x351
github.com/puppetlabs/wash/plugin.cachedOp(0x27aadd0, 0x4, 0x2b01da0, 0xc0074884d0, 0x37e11d600, 0xc0072e7ce0, 0xc007d9b688, 0x100b965, 0x25f85e0, 0x26c89e0)
	/Users/michaelsmith/puppetlabs/wash/plugin/cache.go:190 +0x16f
github.com/puppetlabs/wash/plugin.cachedActionOp(0x25f8500, 0x2b01da0, 0xc0074884d0, 0xc0072e7ce0, 0xc0074884d0, 0x25ff4c0, 0x26c89e0, 0x0)
	/Users/michaelsmith/puppetlabs/wash/plugin/cache.go:167 +0xb3
github.com/puppetlabs/wash/plugin.CachedList(0x2aeaae0, 0xc0072e7cb0, 0x47775a0, 0xc0074884d0, 0xc0074884d0, 0xc0074884d0, 0xc0072e7c01, 0x241f400, 0x2a59870)
	/Users/michaelsmith/puppetlabs/wash/plugin/cache.go:106 +0xc4
github.com/puppetlabs/wash/fuse.(*dir).children(0xc003e62f40, 0x2aeaae0, 0xc0072e7cb0, 0x2a59870, 0x2422e00, 0xc003dd8c10, 0x2aeaae0, 0xc0072e7cb0)
	/Users/michaelsmith/puppetlabs/wash/fuse/dir.go:55 +0xca
github.com/puppetlabs/wash/fuse.(*dir).Lookup.func1(0xc003e62f40, 0xc003d9ddd0, 0x8, 0xc000212050)
	/Users/michaelsmith/puppetlabs/wash/fuse/dir.go:82 +0xb6
created by github.com/puppetlabs/wash/fuse.(*dir).Lookup
	/Users/michaelsmith/puppetlabs/wash/fuse/dir.go:81 +0x6e2
exit status 2
```
when trying to list `mnt/aws/default/resources/s3`.

--- a/plugin/aws/s3Bucket.go
+++ b/plugin/aws/s3Bucket.go
@@ -47,6 +47,10 @@ func listObjects(ctx context.Context, client *s3Client.S3, bucket string, prefix
 
 	for i, p := range resp.CommonPrefixes {
 		prefix := awsSDK.StringValue(p.Prefix)
+		// Skip the top-level '/' prefix, it would be redundant to list it.
+		if prefix == "/" {
+			continue
+		}
 		entries[i] = newS3ObjectPrefix(bucket, prefix, client)
 	}
 

--- a/plugin/entryBase.go
+++ b/plugin/entryBase.go
@@ -2,6 +2,7 @@ package plugin
 
 import (
 	"flag"
+	"fmt"
 	"strings"
 	"time"
 )
@@ -46,7 +47,7 @@ func NewEntry(name string) EntryBase {
 		panic("plugin.NewEntry: received an empty name")
 	}
 	if strings.Contains(name, "/") {
-		panic("plugin.NewEntry: received a name containing a /")
+		panic(fmt.Sprintf("plugin.NewEntry: received name %v containing a /", name))
 	}
 
 	return newEntryBase(name)

